Movies Time/Libraries/NetworkProvider.swift

```diff
@@ -88,7 +109,7 @@ final class NetworkProvider {
         }.resume()
     }
     
-    private func urlFor(requestType: RequestType) -> URL? {
+    private func urlFor(requestType: RequestType, _ id: Int=0) -> URL? {
```

Comment:
as the id is not going to be required always it must be optional  defaulted to nil